### PR TITLE
Portfolio pop up

### PR DESCRIPTION
**Pull request by PierreBarba**
### portfolio PopUp

##General requirements

>  Make sure that there are no linter errors
>  Make sure that you used correct 
>  Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
>  HTML/CSS & JavaScript requirements
>  Follow our list of best practices for HTML & CSS
>  Follow our list of best practices for JavaScript

##  Project requirements

> You` should implement the following interactions:
>  When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
> When the user clicks (or taps) the close (X) button, the popup disappears.

- In order to associate each project with the popup window details, you must refactor the project section:
- You need to use a JavaScript array to store all of the information for all projects.
- For each project you need to store the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, link to source.
- You need to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.
- After that, you could implement the popup window.
- You should implement popups for both mobile and desktop screen sizes

## How do you create my project

- Create a new branch.

- Add my index.js.

- I made the objects.

- Pass the html code to javascript.

- Download figma all resources.

- I gave them functionality.

- I gave them style.

- Create functions to make it appear and disappear.

- Modify the different screens.

- Modify the readme.

- Install linters.

- Fix linter errors.

- Modify css and js to give it the finishing touches.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -114,7 +191,7 @@ body {
 .myRecentWork {
   margin: 22px;
   display: grid;
-  gap: 24px;
+  gap: 12px;
   justify-items: center;
 }
```
Comment (Lizdev-05) [inline style.css:197]: [OPTIONAL]Great job @PierreBarba 😊. Kindly give your card containers padding to create space around the containers and also to meet up with the Figma design.

Also, kindly make the `post` cards take the full width of their mother container in accordance with the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=jv56JbMIOvKnWOOj-0) design.

| Your Design |Figma Design |
| :---:   | :---: | 
| image|image|

File: index.js [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+    ulCard.appendChild(listItem);
+  });
+
+  //   // creando boton
+  const buttonCardEnd = document.createElement('button');
+  buttonCardEnd.type = 'button';
+  buttonCardEnd.classList.add('buttongreen');
+  buttonCardEnd.innerText = 'See Project';
+  buttonCardEnd.setAttribute('onclick', `createModal('${project.key}')`);
+
```
Comment (Lizdev-05) [inline index.js:144]: Kindly check the `See Project` button. It is not working when clicked. The button should listen to the click event according to the requirement and display the project details.
image

**Review by Lizdev-05 — CHANGES_REQUESTED** [review]
Hi @PierreBarba, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- No linter errors 👍
- Good commit messages 👍

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- Kindly remove the unrequited text from your app 👇
image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (PierreBarba) [reply style.css:200]: Thanks for the optional change.

**Review by Tadesse-Alemayehu — APPROVED** [review]
### Hi @PierreBarba 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
68747470733a2f2f6669636f666f72756d732e6d796669636f2e636f6d2f74352f696d6167652f736572766572706167652f696d6167652d69642f343932363069463839303835363833444139334135422f696d6167652d73697

## Highlights
 - [ ] Good job implementing the requested changes.💯
 - [ ]  Correct GitHub Flow 👍🏾
 - [ ]  All linters passed 🥇
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**